tests/test/system/test_exec_graph.cpp

```diff
@@ -0,0 +1,88 @@
+#include <catch2/catch.hpp>
+
+#include "faasm_fixtures.h"
+
+#include <faabric/runner/FaabricMain.h>
+#include <faabric/scheduler/ExecGraph.h>
+#include <faabric/scheduler/Scheduler.h>
+#include <faaslet/Faaslet.h>
+
+namespace tests {
+class ExecGraphTestFixture : public FunctionExecTestFixture
+{
+  public:
+    ExecGraphTestFixture()
+      : m(std::make_shared<faaslet::FaasletFactory>())
+    {
+        // Modify system config (network ns requires root)
```

Comment:
I'm not sure we need this fixture, can we reuse one we already have (e.g. `FunctionExecTestFixture`)?

The `callFunctionGetResult` method seems unnecessary as the result isn't used AFAICT. Can we just use `execFunction`?

This comment also doesn't seem relevant, is it a copy-paste error?

Reply:
You are right, the fixture was unnecessary. The call to `sch.callFunction` is necessary to update the exec graph through faabric (thus we can't use `execFunction` which invokes the function directly on the module).